README.md

```diff
@@ -1,4 +1,4 @@
-# DENO SCRIPTS
+# DENODARSE
 
 [![Deno](https://github.com/rodbecc/deno-scripts/actions/workflows/deno.yml/badge.svg?event=pull_request)](https://github.com/rodbecc/deno-scripts/actions)
 
```

Comment:
what do you think about removing the image here? I guess this doesn't make sense anymore, what do you think? We can open an issue here to list all the scripts automatically. I guess is something simple that could be a good first PR for @thoffart

Reply:
More than that, I guess this badge isn't working now ):

Reply:
> what do you think about removing the image here? I guess this doesn't make sense anymore, what do you think? We can open an issue here to list all the scripts automatically. I guess is something simple that could be a good first PR for @thoffart
The image is from `denon`, but it still lists the scripts, despite of the fact it is showing outdated information. But yeah, it would be nice to have an automated script to show all available scripts. It would be nice to separate the list of production scripts (all of them that is not "flagged" as internal) from the development scripts too (all of them), to avoid duplicity and confusion by newcomers.

> More than that, I guess this badge isn't working now ):
Yeah, I set it to respond on `pull_request` events, since it is more useful than on 'main' merge, but it isn't work. I'm investigating it right now.